File: samps/contests/mini/config/sumit/submissions/time_limit_exceeded/sumitTLE.pl

#
# File:    sumit.pl
# Purpose: to sum the integers from input from stdin
# Author:  pc2@ecs.csus.edu or http://www.ecs.csus.edu/pc2
#
# Tue Nov  1 17:46:37 PDT 2016
# * TODO cause tle by sleeping/running for a long time
#

	$sum = 0;

	while (<>)
	{
		chomp;
		$num = 0 + $_;
		$sum += $num if $num > 0;

	}

	$sum += 1000;

	print "The sum of the integers is $sum\n";

# eof 

File: samps/contests/mini/config/sumit/submissions/time_limit_exceeded/sumitTLE.cs
/*
 * File:    isumit.cs
 * Purpose: to sum the positive from stdin
 * pc2@ecs.csus.edu at http://www.ecs.csus.edu/pc2
 *
 * $Id: hello.cs 3159 2015-09-22 20:39:11Z boudreat $
 * TODO cause tle by sleeping/running for a long time
*/
using System;
 
public class iSumit
{
    static public void Main ()
    {
        int sum = 0;
        string str;
        while((str = Console.ReadLine()) != null) 
        {
            int num = int.Parse(str);
            if (num > 0)
            {
                sum = sum + num;
            }
        }

		sum = sum + 1000;
 
        Console.WriteLine ("The sum of the integers is {0}",sum);
    }
}
